conans/client/profile_loader.py

```diff
@@ -90,7 +90,9 @@ def valid_path(_profile_path):
     if os.path.isabs(profile_name):
         return valid_path(profile_name)
 
-    if profile_name[:2] in ("./", ".\\"):  # local
+    # relative local profile path
+    if profile_name[:2] in ("./", ".\\") or \
```

Comment:
why not use `os.path.isabs` instead of our own bicycles?
that will end up in many misdetected cases, e.g. in posix `..\\myfile` is valid file name

Reply:
Yes, makes total sense, I gonna improve it. Thanks.

Reply:
Done.